src/components/Tabs/Tabs.stories.tsx

```diff
@@ -82,3 +82,33 @@ export const RoutableTabs: Story = () => {
     </MemoryRouter>
   );
 };
+
+export const TabSizes: Story = () => (
+  <>
+    <Tabs selectedValue="one" size="h2" onSelectTab={action('Select Tab')}>
```

Comment:
Do we have a story with no size defined? Just so visual regression covers the default case

Reply:
@vaclav-zeman good point, will add. I guess we could even split every size into separate stories. Maybe Radek will have some ideas too.

Reply:
@vaclav-zeman actually, I've just realized that the first story "Only Tabs" is without size (defaults to h3). Is it okay like that?

Reply:
yep, for sure 👍